backend/config/corpora-api.yml

```diff
@@ -851,8 +849,18 @@ components:
           $ref: "#/components/schemas/collection_uuid"
         collection_visibility:
           $ref: "#/components/schemas/visibility"
+        is_primary_data:
+          type: boolean
+        cell_type:
+          $ref: "#/components/schemas/ontology_element_array"
         cell_count:
           type: integer
+        X_normalization:
+          type: string
+        X_approximate_distribution:
+          $ref: "#/components/schemas/X_approximate_distribution"
+        schema_version:
+          type: string
```

Comment:
should this be enum? We only have a few versions and they change rarely, right?

Reply:
The specification requires this to be a `string`. Moving to an enum would require to change the specs and at this point I don't think it's worth it. That said, @MDunitz could have a stronger opinion on this.

Reply:
We currently only have a few versions but its not clear to me what the cadence of change will be in the future. I dont think there is much to gain from making it an enum (but I could be wrong)

Reply:
If it's informational then a string is fine. If its use goes beyond that, an enum would help avoid certain bugs. Happy to leave it as a string for now if we don't know